src/components/LoadingShape/readme.md

```diff
@@ -0,0 +1,182 @@
+##### LoadingShape solid
+
+```js
+const style = { 
+    width: '400px', 
+    display: 'flex', 
+    justifyContent: 'space-evenly', 
+};
+const circleStyle = { width: '50px', height: '50px' };
+const squareStyle = { height: '50px' };
+const rectStyle = { height: '30px', width: '100px', };
+const roundedStyle = { height: '30px', width: '100px', };
+
+<div className="rainbow-p-vertical_x-large rainbow-p-horizontal_small rainbow-m_auto" style={style}>
+    <div className="rainbow-flex rainbow-flex_column rainbow-justify_end">
+        <div className="rainbow-m-vertical_medium" style={circleStyle}>
+            <LoadingShape shape="circle" />
+        </div>
+        <strong className="rainbow-m-vertical_xx-small">circle</strong>
```

Comment:
The text on the InteractiveExample should change with the theme
<img width="1351" alt="Screen Shot 2020-10-04 at 12 19 17 AM" src="https://user-images.githubusercontent.com/32083556/95007633-4fceea00-05d7-11eb-81ae-7190af3b93e9.png">

Reply:
Fixed